__tests__/health.controller.test.ts

```diff
@@ -0,0 +1,16 @@
+import app from '../src/config/express';
+import * as request from 'supertest';
+
+describe('GET /v1/health', () => {
+  beforeAll((done: jest.DoneCallback) => {
+    done();
+  });
+
+  afterAll((done: jest.DoneCallback) => {
+    done();
+  });
+  it('health api request', async () => {
```

Comment:
Small nitpick, but I think the first parameter to `it(` should be a verb. So this should be `it('gets an OK response from the health api', ...` or something similar.